docs/components/BarChartDocs.js

```diff
@@ -89,9 +48,27 @@ const BarChartDocs = React.createClass({
     });
   },
 
+
   render () {
     const styles = this.styles();
 
+    const ticks = chartData.map(d => {
```

Comment:
Are we cool with one letter function parameters? I am, especially in one liners like this.